packages/sw-routing/src/lib/express-route.js

```diff
@@ -66,8 +74,8 @@ class ExpressRoute extends Route {
    *        HTTP method. Defaults to `'GET'` if not specified.
    */
   constructor({path, handler, method}) {
-    if (path.substring(0, 1) !== '/') {
-      throw ErrorFactory.createError('express-route-requires-absolute-path');
+    if (!(path.startsWith('/') || path.startsWith('http'))) {
+      throw ErrorFactory.createError('express-route-invalid-path');
     }
```

Comment:
I switched things over to check for a prefix of `'http'` rather than `'https://'`. @gauntface's comment was on an earlier incarnation of this PR. Does this look okay now?